### PR TITLE
Reusability

### PR DESCRIPTION
**Pull request by piyush110786**
basic functionality was reused to avoid redundant code.

### CODE REVIEW

File: aasemble/django/apps/api/tests.py [hunk 1]
```diff
@@ -76,42 +76,35 @@ def test_create_repository_incorrect_auth_fails_401(self):
...
-                           'name': 'testrepo',
-                           'binary_source_list': 'deb http://127.0.0.1:8000/apt/eric/testrepo aasemble main',
-                           'source_source_list': 'deb-src http://127.0.0.1:8000/apt/eric/testrepo aasemble main',
-                           'self': response.data['self'],
-                           'sources': response.data['self'] + 'sources/',
-                           'user': 'eric',
-                           'key_id': u''}
-
-        self.assertEquals(response.data, expected_result)
+        if(user == 'eric'):
```
Comment (sorenisanerd) [inline aasemble/django/apps/api/tests.py:87]: "if user == eric" is more pythonic.

Comment (sorenisanerd) [inline aasemble/django/apps/api/tests.py:87]: I don't understand this conditional, btw. Why should this only happen for Eric?

**Comment by piyush110786** [comment]
It was previously written and there are conditionally check that only belong to eric(as previously written so i am not sure needed or not :-) ) but not necessarily needed for other user who create the repository during test case, so I tried not to change previous functionality and just added a check.

**Comment by piyush110786** [comment]
@sorenh what do you say.. do i need to keep this check for eric or just remove it ?? :-)

**Comment by sorenisanerd** [comment]
I can't think of any reason why it should only happen for Eric. Can we just perform the check for everyone?

**Comment by piyush110786** [comment]
@sorenh sure, i will make the changes :-)

**Comment by sorenisanerd** [comment]
Superseded by #136

**Pull request closed without merging**